### PR TITLE
Fixed teardown and launch

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Task: [AB#13425](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13425)
Activity: [AB#13202](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13202)

### CODE REVIEW

File: src/scripts/pscore/tear-down-local.ps1 [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
-    helm delete bc -n buyingcatalogue
+    write-host "Buying Catalogue not installed"
 }
+
+if (kubectl describe pvc -l app=redis -n buyingcatalogue){
+    kubectl delete pvc -l app=redis -n buyingcatalogue
+}
+else {
+    write-host "Redis Cache not found"
+}
```
Comment (bjss-chris) [inline src/scripts/pscore/tear-down-local.ps1:21]: Just needs a trailing newline. 😃

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Other than the missing trailing newline it looks fine to me. However, might be worth @PhysicsJon casting his eye over this as well if he can spare some time as he often spots things that I miss in these scripts.

File: src/scripts/pscore/tear-down-local.ps1 (hunk 1 above)
Comment (gavinwakeley1) [reply src/scripts/pscore/tear-down-local.ps1:21]: Doh! Thought I'd checked that - thanks Chris. Correcting now...

**Review by bjss-chris — APPROVED** [review]
Looks good to me but happy if you want to wait until @PhysicsJon has had a chance to look at it.

**Pull request merged**